pytest_tests/testsuites/container/test_container.py

```diff
@@ -86,6 +87,47 @@ def test_container_creation(self, default_wallet, name):
                 wallet, cid, shell=self.shell, endpoint=self.cluster.default_rpc_endpoint
             )
 
+    @allure.title("Not owner and not trusted party can NOT delete container")
+    def test_only_owner_and_trusted_party_proved_can_delete_container(self, not_owner_wallet, default_wallet):
+        placement_rule = "REP 2 IN X CBF 1 SELECT 2 FROM * AS X"
+        cid = create_container(
+            wallet=default_wallet,
+            rule=placement_rule,
+            shell=self.shell,
+            endpoint=self.cluster.default_rpc_endpoint,
+        )
+
+        with allure.step("Try to delete container"):
+            try:
+                delete_container(
+                    wallet=not_owner_wallet,
+                    cid=cid,
+                    shell=self.shell,
+                    endpoint=self.cluster.default_rpc_endpoint,
+                    await_mode=True,
+                )
+            except Exception as e:
```

Comment:
What if no exception occurs? Will the test fail?

Reply:
> Will the test fail?

Yes, the test will fall

Reply:
> What if no exception occurs? Will the test fail?

Fixed